packages/ui/src/components/sign-up/steps/start.tsx

```diff
@@ -144,10 +146,13 @@ export function SignUpStart() {
 
                   {options.socialButtonsPlacement === 'bottom' ? connectionsWithSeperator.reverse() : null}
 
+                  {!(hasConnection && hasIdentifier && legalConsentEnabled) && <LegalAcceptedField />}
+
                   {userSettings.signUp.captcha_enabled ? <SignUp.Captcha className='empty:hidden' /> : null}
                 </Card.Body>
-                {hasConnection || hasIdentifier ? (
+                {hasIdentifier ? (
                   <Card.Actions>
+                    {legalConsentEnabled && hasIdentifier && <LegalAcceptedField />}
```

Comment:
Similar to the continue step, I feel like this should live within the body not the actions.

Reply:
This is how we render it on AIO right now, it's closer to the `Continue` button when sign up has identifier, when not having identifier and only having social buttons we put this closer to the social button (within the body), if this feels a bit close to the continue button it's probably because we need to adjust the gap

Reply:
If this still does not look right to you, we can bring Alvish to take a look and after that I will make the appropriate changes

Reply:
ok, either way lets make it consistent. Looks like in the continue step its placed in the body.

Reply:
Yeah it's different there though as we wanted to have it on the body, because it's one of the missing fields and it didn't looked right when we were looking the designs with Alvish